rtt/transports/corba/OperationInterface.idl

```diff
@@ -125,17 +137,18 @@ module RTT
      */
     interface COperationInterface
     {
-	typedef sequence<string> COperationList;
+      typedef sequence<string> COperationList;
```

Comment:
Had to adapt rock's ruby bindings to this PR for my test in https://github.com/orocos-toolchain/rtt/pull/122

Hab to switch to COperationDescriptions from COperationList and found that the COperationList typedef here is still defined but not used anymore.